src/main.sh

```diff
@@ -26,6 +27,7 @@ do
   # shellcheck disable=SC2086
   [ -n "${!postinstall}" ] && eval ${!postinstall}
 
+  cd $BASEDIR || exit
```

Comment:
**[shellcheck]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Double quote to prevent globbing and word splitting. [SC2086](https://github.com/koalaman/shellcheck/wiki/SC2086)